unittests/CppInterOp/FunctionReflectionTest.cpp

```diff
@@ -589,6 +589,34 @@ TEST(FunctionReflectionTest, InstantiateTemplateMethod) {
   EXPECT_TRUE(TA1.getAsType()->isIntegerType());
 }
 
+TEST(FunctionReflectionTest, InstantiateVariadicFunctionTemplate) {
+  std::vector<Decl*> Decls;
+  std::string code = R"(
+template<typename... Args> void VariadicFnTemplate(Args... args) {}
```

Comment:
Can we include the example of #305?

Reply:
The current test mimics the same variadic template, I am extending it to initialize using a template pack which should cover the other branch for codecov

Reply:
Still worth adding the test from the issue and adding code to support it.